package.json

```diff
@@ -23,11 +23,11 @@
     "flow-parser": "0.38.0",
     "get-stdin": "5.0.1",
     "glob": "7.1.1",
+    "jest-validate": "18.2.0",
```

Comment:
Got added to the src https://github.com/jlongster/prettier/blob/66fe2f78fa99789e9f784149381ff60265754481/src/options.js#L3